exonum/src/node/consensus.rs

```diff
@@ -634,6 +634,10 @@ impl NodeHandler {
         };
 
         let snapshot = self.blockchain.snapshot();
+        for plugin in &self.plugins {
```

Comment:
Nit: Not sure if this should be called from the consensus code.

Reply:
This is exactly [where the equivalent call was](https://github.com/exonum/exonum/blob/0f622c7b0787f7c8947421d8f75bccb2334c548e/exonum/src/node/consensus.rs#L575) for the explorer before it was moved out.

Reply:
I see. Probably we should later move all the code that is not strictly tied to the consensus to somewhere else. For now it's fine to left things as-is, I guess.